Common/crypto/PkiObjectPsa.c

```diff
@@ -390,7 +390,7 @@ int32_t lWritePublicKeyToPSACrypto( psa_key_id_t xPubKeyId,
 
         psa_set_key_type( &xKeyAttributes, PSA_KEY_TYPE_ECC_PUBLIC_KEY( xKeyFamily ) );
 
-        psa_set_key_algorithm( &xKeyAttributes, PSA_ALG_ECDSA( PSA_ALG_ANY_HASH ) );
+        psa_set_key_algorithm( &xKeyAttributes, PSA_ALG_ECDSA( PSA_ALG_SHA_256 ) );
```

Comment:
This should be fixed in the pal, not here.

Reply:
Moved it to PAL.